openslides_backend/action/actions/committee/update.py

```diff
@@ -21,5 +27,69 @@ class CommitteeUpdateAction(UpdateAction):
             "manager_ids",
             "forward_to_committee_ids",
             "receive_forwardings_from_committee_ids",
+            "organisation_tag_ids",
         ]
     )
+
+    def check_permissions(self, instance: Dict[str, Any]) -> None:
+        if has_organisation_management_level(
+            self.datastore, self.user_id, OrganisationManagementLevel.SUPERADMIN
+        ):
+            return
+
+        manager_ids = None
+        if "manager_ids" in instance:
+            manager_ids = instance["manager_ids"]
```

Comment:
This is the deprecated definition of committee managers. The new permission is defined by the field `committee_$_management_level` (see the wiki for further information). The changes are not fully implemented in the backend yet, but that should not bother you since you can set the data independently of that in the tests.

Reply:
The field `committee_$_management_level` is not in the user model. I cannot use it for tests then. The generate models script is not working.

Reply:
It was introduced in the models.yml a few days ago: https://github.com/OpenSlides/OpenSlides/pull/6029

I think you have to re-generate your models.py